packages/appeals-service-api/src/routes/token-v2.spec.yaml

```diff
@@ -0,0 +1,83 @@
+paths:
+  /api/v2/token:
+    put:
+      tags:
+        - token
+      description: create or update a token for an appeal/user
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - action
+              properties:
+                action:
+                  type: string
+                  example: confirmEmail
+              oneOf:
+                - properties:
+                    id:
+                      description: appeal id
+                      type: string
+                - properties:
+                    email:
+                      description: email of user
+                      type: string
+                      example: test@example.com
+      responses:
+        200:
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+        404:
+          description: Appeal/User not found
+          schema:
```

Comment:
Sorry this is my fault as this is probably copied, this doesn't work and needs the content + type bit.